src/commands/utility/packagist.js

```diff
@@ -0,0 +1,48 @@
+const { SearchCommand, SwitchbladeEmbed, Constants } = require('../../')
+const moment = require('moment')
+
+module.exports = class Packagist extends SearchCommand {
+  constructor (client) {
+    super({
+      name: 'packagist',
+      category: 'utility',
+      parameters: [{
+        type: 'string',
+        full: true,
+        fullJoin: '-',
+        missingError: 'commands:packagist.noNameProvided'
+      }],
+      requirements: { apis: ['packagist'] },
+      embedColor: Constants.PACKAGIST_COLOR,
+      embedLogoURL: 'https://i.imgur.com/MLTCgKN.png'
+    }, client)
+  }
+
+  async search (_, query) {
+    const { data } = await this.client.apis.packagist.search(query)
+    var queryArray = Object.keys(data.results).map(i => data.results[i])
+    return queryArray
```

Comment:
```suggestion
    return Object.keys(data.results).map(i => data.results[i])
```

Reply:
@metehus Requested Changes have been made. You may review it. Thanks